README.md

```diff
@@ -11,13 +14,16 @@ This plugin bring the magic of Giphy translate to Mattermost. For a stable produ
  - [5. Help](#5-help).
  
 ### Requirements
-- For Giphy Plugin 1.0, Mattermost Server v5.14+ is required
-- Giphy plugin requires configuring a [Giphy API Key](https://developers.giphy.com/faq). 
+
+- For Giphy Plugin 1.0, Mattermost Server v5.16+ is required
+- Giphy plugin requires configuring a [Giphy API
+  Key](https://developers.giphy.com/faq). 
 
 ## 1. Users
 
-The giphy plugin implements a `/giphy` command which can be used to look up and post images for phrases.
-It uses the [Giphy Translate API](https://developers.giphy.com/docs/api/endpoint#translate).
+The giphy plugin implements a `/giphy` command which can be used to look up and
```

Comment:
Capitalize the first occurrence of 'giphy' to be consistent with the rest of the README